addons/web/static/src/xml/base.xml

```diff
@@ -505,6 +505,25 @@
     </Portal>
 </t>
 
+<t t-name="OwlPopover" owl="1">
+    <div t-on-click="openPopover">
```

Comment:
The Portal implementation implies that event happening in the ejected part will be redirected within its parent dom tree.

ie, when your Popover is displayed, if a click happens on a the <div class=o_popover/>, the on-click=openPopover will be triggered.

Reply:
true, but to avoid that i would need another node around the popover's `default` slot and i'd rather avoid that as it will force devs to add css nearly everytime they need a popover.

Since the state the call to `openPopover` will do nothing when the popover is already open, i don't think it's such an issue...